PortfolioApp/Users/Views/UserCell/UserViewCell.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+final class UserViewCell: UITableViewCell {
+    @IBOutlet private weak var userNiknameLabel: UILabel!
+    @IBOutlet private weak var emailLabel: UILabel!
+    @IBOutlet private weak var phoneLabel: UILabel!
+    @IBOutlet private weak var websiteLabel: UILabel!
+    @IBOutlet private weak var companyLabel: UILabel!
+    @IBOutlet private weak var addressLabel: UILabel!
+    @IBOutlet private weak var userNameText: UILabel!
+    @IBOutlet private weak var userNiknameText: UILabel!
+    @IBOutlet private weak var emailText: UILabel!
+    @IBOutlet private weak var phoneText: UILabel!
+    @IBOutlet private weak var websiteText: UILabel!
+    @IBOutlet private weak var companyText: UILabel!
+    @IBOutlet private weak var addressText: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {}
+    
+    func setup(with user: User) {
```

Comment:
Static texts insert can be done just once (on awake from nib for example). Dynamic texts should update each time when `setup` method calls as it now